### ISSUE TITLE
When using -zip experiencing errors on a file from OPF Format Corpus

### ISSUE THREAD

**Issue by ross-spencer** [openpreserve/fido#72]
Attempting to scan the opt-format-corpus I'm seeing an error from a specific file: 

    pdfCabinetOfHorrors/embedded_video_quicktime.doc

      goatslayer@goatslayer-acer-linux:~/git/opf-format-corpus/format-corpus/pdfCabinetOfHorrors$ fido -zip embedded_video_quicktime.doc
      FIDO v1.3.3 (formats-v84.xml, container-signature-20160121.xml, format_extensions.xml)
      bad repeat interval
      bad repeat interval
      OK,250,fmt/111,"OLE2 Compound Document Format","OLE2 Compound Document Format",26624,"embedded_video_quicktime.doc","None","signature"
      Traceback (most recent call last):
        File "/usr/local/bin/fido", line 9, in <module>
          load_entry_point('opf-fido==1.3.3', 'console_scripts', 'fido')()
        File "/usr/local/lib/python2.7/dist-packages/opf_fido-1.3.3-py2.7.egg/fido/fido.py", line 855, in main
          fido.identify_file(file)
        File "/usr/local/lib/python2.7/dist-packages/opf_fido-1.3.3-py2.7.egg/fido/fido.py", line 400, in identify_file
          self.identify_contents(filename, type=self.container_type(matches))
        File "/usr/local/lib/python2.7/dist-packages/opf_fido-1.3.3-py2.7.egg/fido/fido.py", line 418, in identify_contents
          raise RuntimeError("Unknown container type: " + repr(type))
      RuntimeError: Unknown container type: 'ole'

Distro stats: 

    Python 2.7.6
    No LSB modules are available.
    Distributor ID:	Ubuntu 
    Description:	Ubuntu 14.04.4 LTS
    Release:	14.04
    Codename:	trusty

My mirror of the OPF Format Corpus can be found here: https://github.com/ross-spencer/opf-format-corpus

**Comment by mistydemeo** [comment]
I see the cause of this one; I introduced this bug by accident.

The code [here](https://github.com/openpreserve/fido/blob/1916c7086e5a3636ee3d67eee8c2ea3e73c533e7/fido/fido.py#L399-L400) branches if the `-zip` option is selected:

```python
if self.zip:
    self.identify_contents(filename, type=self.container_type(matches))
```

The `type` parameter is set to the type of the container, or `None` if the container type can't be detected by FIDO. Prior to 1b5698aa, the only two values were `zip` and `tar`, but that added support for `ole` containers as well. FIDO uses this same method to determine container types for the purpose of identifying single files using container signatures, and for recursing into container files using `-zip`. It's the latter case that's breaking, since `ole` files are causing FIDO to try to unzip the files and identify the contents.

**Comment by mistydemeo** [comment]
Fixed by #73. I can confirm after that PR that `-zip` still works as expected on ZIP files, but no longer breaks on OLE files.

**Comment by sevein** [comment]
Thanks @mistydemeo! @jhsimpson is going to merge #73 soon.
